libraries/AP_Common/AP_Time.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * AP_Time.h
+ */
+
+#ifndef AP_TIME_H
+#define AP_TIME_H
```

Comment:
Agreed, should be pragma once.
On May 19, 2016 6:24 AM, "khancyr" <notifications@github.com> wrote:

> In libraries/AP_Common/AP_Time.h
> <https://github.com/ArduPilot/ardupilot/pull/4163#discussion_r63876702>:
>
> > @@ -0,0 +1,18 @@
> > +/*
> > + * AP_Time.h
> > + */
> > +
> > +#ifndef AP_TIME_H
> > +#define AP_TIME_H
>
> Prefere #pragma once instead of include gard to match with other files
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ArduPilot/ardupilot/pull/4163/files/fcd2a99db974b1fe70dd516b86d6e3af093a1198#r63876702>
>